acvm-repo/acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -377,14 +377,19 @@ impl<F> BlackBoxFuncCall<F> {
 impl<F: Copy> BlackBoxFuncCall<F> {
     pub fn get_inputs_vec(&self) -> Vec<FunctionInput<F>> {
         match self {
-            BlackBoxFuncCall::AES128Encrypt { inputs, .. }
-            | BlackBoxFuncCall::Blake2s { inputs, .. }
+            BlackBoxFuncCall::Blake2s { inputs, .. }
             | BlackBoxFuncCall::Blake3 { inputs, .. }
             | BlackBoxFuncCall::BigIntFromLeBytes { inputs, .. }
             | BlackBoxFuncCall::Poseidon2Permutation { inputs, .. } => inputs.to_vec(),
 
             BlackBoxFuncCall::Keccakf1600 { inputs, .. } => inputs.to_vec(),
-
+            BlackBoxFuncCall::AES128Encrypt { inputs, iv, key, .. } => {
```

Comment:
This was a bug where we were not including all of the AES128Encrypt inputs in inputs vec.

Reply:
There is an `aes128_encrypt` parser test.